### DISCUSSION TITLE
Cybo-Drummer v0.1.0

### DISCUSSION THREAD

**Discussion by HLammers** [HLammers/cybo-drummer#1] [Announcements]
Very first public beta of Cybo-Drummer

<hr /><em>This discussion was created from the release <a href='https://github.com/HLammers/cybo-drummer/releases/tag/'>Cybo-Drummer v0.1.0</a>.</em>

**Comment by bixb922** [comment]
Nice!!!

**Comment by diyelectromusic** [comment]
That's looking really great.

If you do get around to (re)designing a new bespoke, combined board, then it might be worth thinking about MIDI TRS sockets rather than DIN.  I keep thinking I ought to have a TRS version of my router(s)!

Looking forward to hearing what people get up to with it.

Kevin

**Reply by HLammers** [reply]
Thanks for the compliment, Kevin!
Changing from DIN to TRS has crossed my mind, but 3 out of four of my drum computers have DIN MIDI connectors, I do have a lot of DIN midi cables laying around and I would have to deal with the two different versions of MIDI TRS, so probably I’ll stick to DIN.
